gamearea.py

```diff
@@ -368,6 +378,25 @@ def __draw_welcome_message(self, context):
         message = _("(And click the star again to stop it)")
         self.show_message(context, message, 24, y)
 
+    def __draw_gameover(self, context):
+        if not self.win:
+            score = self.score - 20
+        else:
+            score = self.score
+        y = 0
+        self.playing = False
+        message = _("Game Over")
+        y = self.show_message(context, message, 124, -100)
+        your_score = "%s %d" % (_("Your Score:"), score)
+        y = self.show_message(context, your_score, 60, 50)
+        high_score = "%s %d" % (_("High Score:"), score)
+        y = self.show_message(context, high_score, 60, 150)
+        message = _("Click on Star to srat the game")
+        
+        self.score = 0
+        self.win = True
+        y = self.show_message(context, message, 24, 250)
```

Comment:
Would be helpful to use a better variable name than y.

Reply:
oh I forgot to remove that variable. Just used it for testing.